service/routes.py

```diff
@@ -182,4 +182,20 @@ def get_item(customer_id, item_id):
         logger.error(f"Customer {customer_id} and corresponding item {item_id} could not be found.")
         abort(status.HTTP_404_NOT_FOUND, f"Customer {customer_id} and corresponding item {item_id} could not be found.")
  
-
+######################################################################
+# DELETE A SHOPCART
+######################################################################
+@app.route("/shopcarts/<int:customer_id>", methods=["DELETE"])
+def delete_shopcart(customer_id):
+    """
+    Delete a shopcart
+    This endpoint will delete an shopcart based on the customer_id specified in the path
+    """
+    app.logger.info("Request to delete shopcart with id: %s", customer_id)
+    shopcart = ShopCarts.find_by_customer_id(customer_id)
+    if shopcart:
+        shopcart.delete()
```

Comment:
this would be a list, so I don't think we can call delete on it directly

Reply:
I changed this to find(), I think it can return a shopcart

Reply:
this function should be deleting multiple records (all items in a shopcart of a customer), so it should return multiple records and delete them all